e2e/tests/oai/002.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+#!/usr/bin/env bash
+# SPDX-license-identifier: Apache-2.0
+##############################################################################
+# Copyright (c) 2023 The Nephio Authors.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Apache License, Version 2.0
+# which accompanies this distribution, and is available at
+# http://www.apache.org/licenses/LICENSE-2.0
+##############################################################################
+
+## TEST METADATA
+## TEST-NAME: Deploy OAI Core Operators
+##
+
+set -o pipefail
+set -o errexit
+set -o nounset
+[[ ${DEBUG:-false} != "true" ]] || set -o xtrace
+
+# shellcheck source=e2e/defaults.env
+source "$E2EDIR/defaults.env"
+
+# shellcheck source=e2e/lib/k8s.sh
+source "${LIBDIR}/k8s.sh"
+
+# shellcheck source=e2e/lib/kpt.sh
+source "${LIBDIR}/kpt.sh"
+
+k8s_apply "$TESTDIR/002-oai-operators.yaml"
+
+for pkgvar in common-regional-database cp-operators up-operators; do
+    k8s_wait_ready "packagevariant" "oai-$pkgvar"
+done
+kpt_wait_pkg "regional" "database"
```

Comment:
@electrocucaracha Instead of regional the core network components, database, amf, smf, nrf, udr, udm and control plane controllers (oai-cp-operators) will be deployed on the `core` cluster not regional. 

On the regional cluster there will be only `CU-CP`